server/queue/queue-rabbitmq/src/test/resources/json/mailqueueview/configuration/configuration_edited.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "cassandra-mail-queue-view-configuration",
+  "eventId": 0,
+  "aggregateKey": "CassandraMailQueueViewConfiguration",
+  "bucketCount": 1,
+  "updateBrowseStartPace": 1000,
+  "sliceWindow": 3600.000000000
```

Comment:
it doesn't look like a great configuration format. What about `1 hour` ?

Reply:
Why keep the unparsed value and bother us with formatting issues?

Reply:
it's not unparsed, it's serialized to a human-readable value.
Why ? Because text format are supposed to be readable, right ?
Otherwise we can just use a binary format.
Anyway, you are free to ignore my comment, I was suggesting something better but we can't always take all comments into account and it's alright.